core/src/com/pockball/pockball/ecs/Engine.java

```diff
@@ -28,7 +28,28 @@
     private final EntityFactory entityFactory;
 
     private World world;
-    private Body walls;
+
+    private final Vector2[][] ballLocations = {
```

Comment:
Looks good, but this could maybe in the future be calculated relative to the center of the table?